tools/installer/requirements.txt

```diff
@@ -1,3 +1,5 @@
+git+git://github.com/exaile/pywin32-ctypes.git@pyinstaller#egg=pywin32-ctypes
+git+git://github.com/dangmai/pyinstaller.git@msys2#egg=PyInstaller
 musicbrainzngs
 mutagen==1.31
```

Comment:
yeah, the integration test suite to detect regressions… I'd like to have that too ;)